frontend/express/public/javascripts/countly/countly.views.js

```diff
@@ -3328,6 +3335,55 @@ window.ManageUsersView = countlyView.extend({
         $(".change-password").off("click").on('click', function() {
             $(this).parents(".row").next().toggle();
         });
+
+        $('body').off('change').on('change', '.pp-uploader', function() {
+            $('.pp-menu-list').hide();
+            var member_id = $(this).data('member-id');
+            $(this).simpleUpload("/member/icon", {
```

Comment:
If we use library in the core, we should better add it to the core and not to the plugin. And it might be useful to other parts too may be.
And don't forget to add it to gruntfile too.

Reply:
added.